spec/controllers/api/v1/book_suggestion_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe API::V1::BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'When creating a valid book suggestion' do
+      let!(:new_book_suggestion_attributes) { build(:book_suggestion).attributes }
```

Comment:
use `attributes_for(:book_suggestion)` instead of building a new book_suggestion.

Reply:
done! but I needed to use the merge method with the attributes that are associations because attributes_for does not create the attribute.